noir-projects/noir-contracts/contracts/app/amm_contract/src/main.nr

```diff
@@ -326,7 +331,6 @@ pub contract AMM {
 
         // We transfer the full amount in, since it is an exact amount, and prepare a partial note for the amount out,
         // which will only be known during public execution as it depends on the live balances.
-        // TODO(#10286): consider merging these two calls
```

Comment:
This TODO was incorrectly placed as the partial note token and the token_in are different.